src/local_sensitivity.jl

```diff
@@ -70,11 +80,15 @@ function ODELocalSensitivityProblem(f::DiffEqBase.AbstractODEFunction,u0,
                                     alg = SensitivityAlg();
                                     callback=CallbackSet(),mass_matrix=I)
   isinplace = DiffEqBase.isinplace(f)
+  isautojacvec = get_jacvec(alg)
```

Comment:
This is dependent on alg? I thought this would be a user-settable switch which defaults to true. We should always try to use the AD seeded version here since it should be much more efficient.

Reply:
Oh nevermind, I see where it's passed